Week1/SantiagoMedina/Gradle/fatJar/body-factory/src/main/java/body/InterfaceBody.java

```diff
@@ -0,0 +1,7 @@
+package vehiclefactory;
+
+public interface InterfaceBody {
+    
+    public void description();
```

Comment:
In software development, names are pretty important and difficult to set. Try to think twice about a name, you might find a better one

Reply:
ok, Is better other name. I changed the name to BodyMethods.